### PR TITLE
An example of custom Galleon layers, covers both plugin and legacy approaches

### PR DESCRIPTION
**Pull request by jfdenise**
The mariadb-galleon-pack project is identical to the output of what we will document to build a custom feature-pack for EAP 8 Beta.
The application project is what we will use to document the custom galleon layers feature usage in an openshift context. Allows to provision with both plugin and legacy approaches based on Maven profiles.

### CODE REVIEW

File: examples/eap/docker-build/README.md [hunk 1]
```diff
@@ -54,10 +54,10 @@ Technologies required to build and deploy this example
 
 * You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
 
-* You have installed the repository for the Helm charts for WildFly
+* You have installed the repository for the Helm charts for EAP 8
 
  ```
-helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+helm repo add jboss https://jbossas.github.io/eap-charts/
```
Comment (jmesnil) [inline examples/eap/docker-build/README.md]: should be `helm repo add jboss-eap https://jbossas.github.io/eap-charts`  as you refer to the Helm Chart with `jboss-eap/eap8`.

File: examples/eap/legacy-s2i-source-build/README.md [hunk 2]
```diff
@@ -65,10 +65,10 @@ Technologies required to build and deploy this example
 
 * You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
 
-* You have installed the repository for the Helm charts for WildFly
+* You have installed the repository for the Helm charts for EAP 8
 
  ```
-helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+helm repo add jboss https://jbossas.github.io/eap-charts/
```
Comment (jmesnil) [inline examples/eap/legacy-s2i-source-build/README.md]: idem

File: examples/eap/s2i-source-build/README.md [hunk 3]
```diff
@@ -50,10 +50,10 @@ Technologies required to build and deploy this example
 
 * You have installed Helm. Please refer to [Installing Helm page](https://helm.sh/docs/intro/install/) to install Helm in your environment
 
-* You have installed the repository for the Helm charts for WildFly
+* You have installed the repository for the Helm charts for EAP 8
 
  ```
-helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+helm repo add jboss https://jbossas.github.io/eap-charts/
```
Comment (jmesnil) [inline examples/eap/s2i-source-build/README.md]: idem

**Review by jmesnil — CHANGES_REQUESTED** [review]

File: examples/eap/docker-build/README.md (hunk 1 above)
Comment (jfdenise) [reply examples/eap/docker-build/README.md]: Thank-you. Fixed.

File: examples/eap/legacy-s2i-source-build/README.md (hunk 2 above)
Comment (jfdenise) [reply examples/eap/legacy-s2i-source-build/README.md]: Thank-you. Fixed.

File: examples/eap/s2i-source-build/README.md (hunk 3 above)
Comment (jfdenise) [reply examples/eap/s2i-source-build/README.md]: Thank-you. Fixed.

**Pull request merged**